modules/core/src/main/java/org/apache/ignite/internal/processors/task/GridTaskProcessor.java

```diff
@@ -1138,14 +1139,15 @@ public void processJobExecuteResponse(UUID nodeId, GridJobExecuteResponse msg) {
         lock.readLock();
 
         try {
+            GridTaskWorker<?, ?> task = tasks.get(msg.getSessionId());
```

Comment:
Why this is not inlined in method argument, like

tryResolveTaskName(tasks.get(msg.getSessionId()) ?

The same for other calls.